registry.go

```diff
@@ -89,7 +89,7 @@ func mapStoreRegistryName(registry string) string {
 	return registry
 }
 
-func mapAuthenticationRegistryName(hostname string) string {
+func MapRegistryNameRegistry1DockerIo(hostname string) string {
```

Comment:
The method name is not meaningful.

Reply:
This method maps a host name to a key for credential store. Therefore, `func ServerAddressFromHostname(hostname string) string` with documentation is preferable.

Reply:
changed accordingly.